framework/console/controllers/BaseMigrateController.php

```diff
@@ -287,7 +288,7 @@ public function actionRedo($limit = 1)
         if (empty($migrations)) {
             $this->stdout("No migration has been done before.\n", Console::FG_YELLOW);
 
-            return self::EXIT_CODE_NORMAL;
+            return ExitCode::OK;
```

Comment:
Maybe it is actual better to use a predefined method in `console\Controller` for `OK` and `Error`:

```php
public function actionFoo()
{
    // ...
    return $this->exitOk();
}
```

To hide constants behind the scenes.

This would be consistent with recent addition of `web\Controller::asJson()` method.